projects/lib-magnetar-quill/src/lib/components/table-modal/table-modal.component.css

```
@@ -0,0 +1,96 @@
+}
+
+.modal-actions {
+  display: flex;
+  justify-content: flex-end;
+  gap: 10px;
+  margin-top: 20px;
+}
+
+.modal-actions button {
+  padding: 8px 16px;
+  border: none;
```

Stylelint (17.14.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)